assessments/projects/external-secrets/self-assessment.md

```diff
@@ -0,0 +1,371 @@
+
+* Compliance - No specific compliance certifications or adherence to standards like PCI-DSS, COBIT, ISO, or GDPR have been documented. 
+
+## Secure Development Practices
+
+ESO has achieved a "passing" Open Source Security Foundation (OpenSSF) best practices badge
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/5947/badge)](https://www.bestpractices.dev/projects/5947). The project is working on receiving a silver badge and is in the process of meeting the criteria for it.
+
+### Development Pipeline
+
+* All source code is publicly maintained in [GitHub](https://github.com/external-secrets/external-secrets).
+* Code changes are submitted via Pull Requests (PRs) and must be reviewed and approved by at least one maintainer.
+* Commits to the `main` branch are merged only when a PR is approved and passes all checks.
+* Once a pull request has been opened it will be assigned to a reviewer from external-secrets/maintainers.
+* ESO uses the GitHub Bot [Paul the Alien](https://github.com/apps/paul-the-alien) (which is still in Alpha) to store the list of maintainers, to approve/merge PRs and to delete branches after PR merges.
+* Raising a PR triggers a series of GitHub actions and workflows whose component checks are broken down below:
+  * [Sonar Cloud Quality Gate](https://sonarcloud.io/project/issues?id=external-secrets_external-secrets) check initiated by the sonarcloud bot which checks for bugs, vulnerabilities, security hotspots, code smells, code coverage and duplication
+  * Developer Certificate of Origin (DCO) check to verify commits are signed correctly
+  * Detect noop ([skip-duplicate-actions](https://github.com/fkirc/skip-duplicate-actions)) 
+  * Linting check
+  * Diff check
+  * Unit tests
+  * E2E tests
+  * Dependency License Checks (FOSSA)
+  * Building the image
+  * Image scanning for vulnerabilities using Trivy
+  * Create SBOM & provenance files and sign the image
+  * Publish signed artifacts to Dockerhub
+
+### Communication Channels
+* Referenced in docs under [How to Get Involved](https://external-secrets.io/latest/#how-to-get-involved) and described below:
+  * Internal:
+    * Bi-weekly Development Meeting every odd week at 8:00 PM Berlin Time on Wednesday ([agenda](https://hackmd.io/GSGEpTVdRZCP6LDxV3FHJA), [jitsi call](https://meet.jit.si/eso-community-meeting))
+    * [Kubernetes Slack: #external-secrets channel](https://kubernetes.slack.com/messages/external-secrets)
+    * [Github Issues](https://github.com/external-secrets/external-secrets/issues)
+    * [Github Discussions](https://github.com/external-secrets/external-secrets/discussions)
+  * Inbound:
+    * [Github Issues](https://github.com/external-secrets/external-secrets/issues)
+    * [Github Discussions](https://github.com/external-secrets/external-secrets/discussions)
+    * [Contributing Process](https://external-secrets.io/latest/contributing/process/)
+    * Contact Email: contact@external-secrets.io
+  * Outbound:
```

Comment:
Is there a mailing list or similar to handle mass communication?

Reply:
None is available publicly. I've reached out to ESO maintainers to address this. Will update here.

Reply:
Update: ESO doesn't use any mailing lists currently.

Reply:
Also, the current email given was outdated. I've updated it with the new email in this document, and intend to follow up on updating it in ESO's Security.md file.

Reply:
This is also done - https://github.com/external-secrets/external-secrets/pull/2949